migrations/2023-12-12-150828_community_aggregates_add_local_subscribers/up.sql

```diff
@@ -0,0 +1,78 @@
+-- Couldn't find a way to put subscribers_local right after subscribers except recreating the table.
+ALTER TABLE community_aggregates
+    ADD COLUMN subscribers_local bigint NOT NULL DEFAULT 0;
+
+-- update initial value
+UPDATE
+    community_aggregates ca
+SET
+    subscribers_local = (
+        SELECT
+            COUNT(*)
+        FROM
+            community_follower cf
+        WHERE
+            cf.community_id = ca.community_id
+            AND (
+                SELECT
+                    local
+                FROM
+                    person
+                WHERE
+                    person.id = cf.person_id));
```

Comment:
This doesn't look right. Read the link I sent again, you shouldn't need a subquery for this. And it needs to be filtering based on local=true.